torch/testing/_internal/common_methods_invocations.py

```diff
@@ -281,6 +282,31 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad):
                                     low=None, high=None,
                                     requires_grad=False))),)
 
+def sample_inputs_linalg_inv(op_info, device, dtype, requires_grad=False):
+    """
+    This function generates always invertible input for torch.linalg.inv using
+    random_fullrank_matrix_distinct_singular_value.
+    The input is generated as the itertools.product of 'batches' and 'ns'.
+    In total this function generates 8 SampleInputs
+    'batches' cases include:
+        () - single input,
+        (0,) - zero batched dimension,
+        (4,) - batch of two matrices,
```

Comment:
of two or four matrices?

Reply:
Let's set it to two matrices, so that's faster 🙂